core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -691,7 +691,7 @@ static void bn_clear(struct bignum *s)
 {
 	struct mpa_numbase_struct *bn = (struct mpa_numbase_struct *)s;
 
-	memset(bn, 0, bn->alloc);
+	memset(bn->d, 0, bn->alloc);
```

Comment:
Please also try to clean-up the commit message abit, the title is rather long, we usually try to follow the [50/72 rule](http://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting) (not 100% strictly, but somewhat in that ball park). Feel free to put RB too:

`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`